README.rst

```diff
@@ -23,8 +23,6 @@ Disclosure
 -  This code has not been through a third party security audit.
 -  I’ve successfully tested this locally with ``pypy-5.4.1``. TravisCI
    has confirmed this doesn’t work with their version.
--  From their logs:
-   ``RuntimeError: PyNaCl is not compatible with PyPy < 2.6. Please upgrade PyPy to use this library.``
 
```

Comment:
Let's add a notice that DCN no longer supports PyNacl and that newer versions of will not be backwards compatible.

I think I'm going to make this a 1.0.0 push - taking her out of beta :-D